package.json

```diff
@@ -54,7 +54,7 @@
         "ng-pick-datetime": "7.0.0",
         "ng-starrating": "1.0.20",
         "ng2-ace-editor": "0.3.9",
-        "ng2-charts": "2.4.3",
+        "ng2-charts": "2.3.0",
```

Comment:
Is there a particular reason for a version decrement?

Reply:
Yes. This is a hotfix.
Everything above 2.3.0 uses `lodash-es` as a dependency while 2.3.0 uses `lodash`. For some reason, it takes ages to import `lodash-es` while it's fast for us to import `lodash` => The reason that the builds hang and Bamboo cancelled them was exactly this import.
Therefore downgrading is the hotfix for the build time issues.
Since this is only a hotfix we are in discussion with Stephan on how to fix this in a long term.
We checked all occurrences of the charts and made sure that they are still rendered as expected.

Reply:
This is the fix for the global client test failures. Don't know if you noticed, but there are some PRs where the client tests "hung up unexpectedly". This is because some test executions take too long. Reason for that is that they import a module called lodash-es. (Test exercutions take like 700s where the lodash-es import is like 680 seconds) version 2.3.0 of ng2-charts is the latest version which does not use lodash-es, but rather lodash which does not produce any problems.